src/nvim/testdir/bench_re_freeze.in

```diff
@@ -0,0 +1,13 @@
+Test for Benchmarking RE engine
```

Comment:
Didn't realize it took 15s, that's a good point. I suggest creating a new directory, `test/benchmark/`. We have other plans for performance metrics and will eventually need this directory anyway.

Then just toss in a new target in the top-level `Makefile`:

```
benchmark: | nvim
▸       +$(BUILD_CMD) -C build benchmark
```

and in `CMakeLists.txt` add this near the bottom:

```
  add_custom_target(benchmark
    COMMAND ${CMAKE_COMMAND}
      -DBUSTED_PRG=${BUSTED_PRG}
      -DNVIM_PRG=$<TARGET_FILE:nvim>
      -DWORKING_DIR=${CMAKE_CURRENT_SOURCE_DIR}
      -DBUSTED_OUTPUT_TYPE=${BUSTED_OUTPUT_TYPE}
      -DTEST_DIR=${CMAKE_CURRENT_SOURCE_DIR}/test
      -DBUILD_DIR=${CMAKE_BINARY_DIR}
      -DTEST_TYPE=benchmark
      -P ${PROJECT_SOURCE_DIR}/cmake/RunTests.cmake
    DEPENDS nvim tty-test)
```

And we'll leave it at that. It doesn't have to be a perfect solution right now, that should be good enough.